content/en/docs/ambient/upgrade/helm/revision_test.sh

```diff
@@ -48,6 +48,8 @@ _rewrite_helm_repo snip_rollback_tag
 
 # upgrading a tag creates an MWC, let's clean it up 
 export REVISION=istio-1-22-1
+export OLD_REVISION=istio-1-21-2
```

Comment:
As mentioned above, you can generate `delete_old_revision` from tthe docs and call it here, so you don't have to hardcode the same commands in 2 spots.